packages/ui5-middleware-cfdestination/README.md

```diff
@@ -10,18 +10,29 @@ npm install ui5-middleware-cfdestination --save-dev
 
 ## Configuration options (in `$yourapp/ui5.yaml`)
 
-- debug: true|false  
+- `debug`: `<bool>`, default: `false`  
```

Comment:
typically we just name it `boolean` but I don't really care - as log as it is consistent in your middleware I am fine

Reply:
`boolean` it is now